CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 0.65.0 (Unreleased)
 
+## 0.64.2
```

Comment:
Please do include it in the Unreleased block. The exact version number will be determined at the release time.